crypto/crypto/src/test_utils.rs

```diff
@@ -98,6 +98,30 @@ where
         .no_shrink()
 }
 
+/// Produces a uniformly random keypair from a seed and the user can alter this sleed slightly.
+/// Useful for circumstances where you want two disjoint keypair generations that may interact with
+/// each other.
+#[cfg(any(test, feature = "fuzzing"))]
+pub fn uniform_keypair_strategy_with_perturbation<Priv, Pub>(
+    perturbation: u8,
```

Comment:
Ooc, it's just a single byte because atm we only have two conflicted key generations and we don't expect more than 256, right?

Reply:
Correct. If we're going to be using more than 256 independently changing the structure of the proptest seems reasonable.